packages/tools/testTools/src/seleniumTestUtils.ts

```diff
@@ -0,0 +1,43 @@
+import type { ThenableWebDriver } from "selenium-webdriver";
+import "selenium-webdriver/safari";
+import { evaluateDisposeSceneForVisualization, evaluateInitEngineForVisualization, evaluatePrepareScene, evaluateRenderSceneForVisualization } from "./visualizationUtils";
+
+// Take Playgorund Id and Selenium webdriver and load snippet info into test page
+export const LoadPlayground = async (
+    driver: ThenableWebDriver,
+    pgId: string,
+    globalConfig: { root: string; baseUrl: string; snippetUrl: string; pgRoot: string },
+    framesToRender: number = 500
+): Promise<void> => {
+    await driver.executeScript(
+        `
+                var _globalConfig = {
+                    'root': '${globalConfig.root}',
+                    'baseUrl': '${globalConfig.baseUrl}',
+                    'snippetUrl': '${globalConfig.snippetUrl}',
+                    'pgRoot': '${globalConfig.pgRoot}}',
+                };
+
+                const evaluateInitEngineForVisualization = ${evaluateInitEngineForVisualization.toString()}
+                const evaluatePrepareScene = ${evaluatePrepareScene.toString()}
+                const evaluateRenderSceneForVisualization = ${evaluateRenderSceneForVisualization.toString()}
+
+                globalThis.testSuccessful = false;
+
+                await evaluateInitEngineForVisualization("webgl1", false, false, "${globalConfig.baseUrl}");
+                await evaluatePrepareScene({playgroundId: "${pgId}"}, _globalConfig);
+                evaluateRenderSceneForVisualization(${framesToRender});
+            `
```

Comment:
I would recommend putting this code into a local function and executing it so that there is syntax checking.